docs/usermanual.md

```diff
@@ -252,7 +252,7 @@ This mechanism and the bidirectionality plugin cannot be simultaneously activate
 ##### Configuration command topic
 
 ```text
-/{{apikey}}/{{deviceid}}/configuration/commands
+/ul/{{apikey}}/{{deviceid}}/configuration/commands
```

Comment:
Is configuration command considered a command? (from the point of view of not-usage of the prefix)

Reply:
Updated in 34b2271
All the topics subscribed by the agent (to send measures, to configuration command retrieval or to get result of a command) are prefixed with the agent procotol (/ul)
All topis published by the agent (to send a comand or to send configuration information) to a device are not prefixed by the protocol, in this case '/ul',